dev-docs/howtoguides/release_a_new_version.md

```diff
@@ -187,6 +187,15 @@ If this is your first time releasing ubuntu-advantage-tools, you'll need to do t
 
     h. Tell the SRU team member who performed the pre-SRU review that the packages are in the -proposed release queue. They will need to actually approve the package to move into -proposed.
 
+    i. Release the documentation changes
+      * Open a pull request bringing commits from the `docs-devel` branch to the `docs` branch. Ask in the ~Pro channel for a review. Some conflicts may apperar but those should be only content-related and easy enough to fix.
+      * Once the pull request lands, the `latest` documentation will be automatically updated.
+      * Tag the HEAD of the `docs` branch as `v<version>`. Example: `v28`.
+        * the `v` in the tag differentiates the documentation releases as they will appear in readthedocs from the software version tags in the `release` branch.
+      * Go to [the project in readthedocs](https://readthedocs.com/projects/canonical-ubuntu-pro-client/), to the `Versions` tab, look for the tag in the inactive versions list and click `Activate`.
+        * Be sure the `Active` checkbox is checked, and the privacy level is set to `Public`.
+      * Merge the `docs` branch back to `docs-devel` if there were are any changes applied only to `docs`.
```

Comment:
Should we use merge commits for the docs branches (as opposed to rebasing)? Or does it not matter/no one cares?

Reply:
hmmm, the difference is a technicality in this case. I personally do not care at all. I don't see pros or cons immediately.

Reply:
All good I won't care either then :)

Reply:
merge it is, just added it there